pars-fmt/src/common.rs

```diff
@@ -0,0 +1,35 @@
+//! Utilities and shared helpers.
+
+/// Fields in a struct or enum member.
+pub enum Fields<'a> {
+    Unnamed(usize),
+    Named(&'a [&'a str]),
+}
+
+impl<'a> From<&'a [&'a str]> for Fields<'a> {
```

Comment:
I feel a little weird about implementing `From` here, but since it's a convenience type, I think it's fine, and I couldn't do it better.